### ISSUE TITLE
Update Homebrew/MacPorts environment for the next maintenance

### ISSUE THREAD

**Issue by Kazzz-S** [Kazzz-S/klayout#36]
## Homebrew ##

On Catalina,

* Jan.2020  ---> as of Feb. 18th 2020
* Ruby 2.6.5 ---> 2.7.0
* Python 3.7.5 ---> 3.7.6
----
```
(base) MacBookPro2{sekigawa}(1)$ brew info ruby
ruby: stable 2.7.0 (bottled), HEAD [keg-only]
Powerful, clean, object-oriented scripting language
https://www.ruby-lang.org/
/usr/local/Cellar/ruby/2.7.0 (20,346 files, 33.1MB)
```
```
(base) MacBookPro2{sekigawa}(3)$ brew info python
python: stable 3.7.6 (bottled), HEAD
Interpreted, interactive, object-oriented programming language
```

**Comment by Kazzz-S** [comment]
## MacPorts ##

```
(base) MacBookPro2{sekigawa}(22)$ port installed python*
The following ports are currently installed:
  python2_select @0.0_3 (active)
  python3_select @0.0_1 (active)
  python27 @2.7.17_0 (active)
  python37 @3.7.6_0 (active)
  python38 @3.8.1_1 (active)
  python_select @0.3_8 (active)
```

**Comment by Kazzz-S** [comment]
If 0.26.4 does not merge 0.26.3-mac1, do merge this, too.
https://github.com/kazzz-s/klayout/commit/5bb85c749007abe6669d90f07d949fb22c4e86e9 

----
DONE!

**Comment by Kazzz-S** [comment]
https://github.com/Kazzz-S/klayout/issues/36#issue-566535966
DONE in version 0.26.4.

**Comment by Kazzz-S** [comment]
Released as https://github.com/KLayout/klayout/issues/519

**Comment by Kazzz-S** [comment]
Homebrew updated Python to 3.7.7 on March 12, 2020.

**This has to be merged into the next maintenance.**

**Comment by Kazzz-S** [comment]
Merged for building 0.26.5.

https://github.com/KLayout/klayout/issues/552
https://github.com/KLayout/klayout/pull/553
